src/OpenTelemetry/Logs/OpenTelemetryLoggerOptions.cs

```diff
@@ -15,12 +15,32 @@
 // </copyright>
 
 #if NETSTANDARD2_0
+using System;
+using System.Collections.Generic;
 using Microsoft.Extensions.Logging;
 
 namespace OpenTelemetry.Logs
 {
     public class OpenTelemetryLoggerOptions
     {
+        public readonly List<LogProcessor> Processors = new List<LogProcessor>();
```

Comment:
Did you intend for this to be public? The readonly does not prevent operations on the list.

Reply:
Good catch, I've changed this to internal.